### DISCUSSION TITLE
About survival mode

### DISCUSSION THREAD

**Discussion by lenocid** [ikemen-engine/Ikemen-GO#929] [Feature Request] [answered]
1.The music will restart every round(Is there a way to loop the music for survival mode throughout all rounds on a single stage? Right now every time a new round begins, the music restarts.)
2.powerpersistence feature.
3.Introduction can be cancel after first match? Just like mugen.
4.Each round is round1. Can you change it to round2, round3...?

**Comment by lenocid** [comment]
I tried to imitate it, but it didn't work.
![Snipaste_2022-10-09_16-03-16](https://user-images.githubusercontent.com/112241338/194758620-6deeecbc-858d-4916-8e94-f71e2379f5b1.png)

**Comment by K4thos** [comment, marked as answer]
1 was a bug that is already fixed
